e2e/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const PRODUCT_NAME = process.env.PLAYWRIGHT_PRODUCT_NAME!;
```

Comment:
Don't force type these. As I mentioned in my earlier comments, these need to be asserted or tested in some other way. They are user input, and we will need to disable tests or fail the whole run if they are not set.

Reply:
Also, please keep the more verbose name intact. It makes it clearer / easier to throw the appropriate error. If this isn't defined and you want to assert / throw you want to throw `PLAYWRIGHT_PRODUCT_NAME is not defined`, not `PRODUCT_NAME is not defined`. The implementer needs to know what the problem is and how to correct it.

Reply:
I understand that, but it didn't pass unit tests